README.md

```diff
@@ -99,17 +99,19 @@ After the import, you will get one of the following results:
    **c. Missing required properties**:
```

Comment:
@aminerhanemi most of these errors are incorrect.

(a), (b) and (c) cannot be returned from the plugin because we don't check anymore the data ingested against the Linkurious schema.

Also (d), with the development of:
https://linkurious.atlassian.net/browse/LKE-4206

is probably obsolete and it will be thrown as a "failed" import status instead of "incomplete"

@renato-rg to confirm.

Reply:
yes we should also remove a, b  and c.
How does lke-4206 prevent (d) ? failing means nothing will be imported